packages/flutter_tools/lib/src/android/android_builder.dart

```diff
@@ -26,9 +26,8 @@ abstract class AndroidBuilder {
   /// Builds an AAR artifact.
   Future<void> buildAar({
     @required FlutterProject project,
-    @required AndroidBuildInfo androidBuildInfo,
-    @required String target,
```

Comment:
This seems unrelated right? We should still keep the ability to specify the main dart file

Reply:
oops. reverted